### DISCUSSION TITLE
How do i host cloud variables on the packager ?

### DISCUSSION THREAD

**Discussion by M1nexx** [TurboWarp/packager#843] [Q&A]
How can I host cloud variables for a multiplayer game, compiled with the packager, intended to be released? (on itch.io or steam)
![image](https://github.com/TurboWarp/packager/assets/134965745/03ba58cd-8266-4157-a2bc-4677f5c5efa9)

**Comment by GarboMuffin** [comment]
Just use "connect to cloud variable server" if you want them to behave like cloud variables

**Reply by M1nexx** [reply]
same with scratch-turbowarp extern host ?

**Reply by GarboMuffin** [reply]
i don't know what you mean by that

it will use the host you type into the box. the default works pretty okay

**Reply by M1nexx** [reply]
okay thanks :)
